example/tutorials/t8_step4_partition_balance_ghost.cxx

```diff
@@ -118,7 +119,7 @@ T8_EXTERN_C_BEGIN ();
 /* In this function we create a new forest that repartitions a given forest
  * and has a layer of ghost elements. 
  */
-static t8_forest_t
+static              t8_forest_t
```

Comment:
```suggestion
static t8_forest_t
```

Reply:
This indentation is caused by the t8indent-script. Shall theses files not be indented using the t8indent-script?

Reply:
indent issues should be all solved, as i merged in the current develop branch.